backend/app/models/prompt_version.py

```
@@ -0,0 +1,51 @@
+    version: int
+    label: PromptVersionLabel = Field(
+        default=PromptVersionLabel.STAGING, nullable=False
+    )
+
+    inserted_at: datetime = Field(default_factory=now)
+    updated_at: datetime = Field(default_factory=now)
+
+    is_deleted: bool = Field(default=False)
+    deleted_at: datetime | None = Field(default=None)
+
+    prompt: "Prompt" = Relationship(back_populates="versions")
```

Ruff (0.12.2)

Undefined name `Prompt`

(F821)